components/hubspot/hubspot.app.mjs

```diff
@@ -0,0 +1,557 @@
+import { axios } from "@pipedream/platform";
```

Comment:
I think so. I didn't use the sdk client lib in this PR because of the [latency/timeout issues ](https://pipedream-users.slack.com/archives/CPTJYRY5A/p1647895750121889)we were having with it, and it doesn't support some endpoints like `/crm/v4/associations`.